<ui version="4.0" >
 <class>CostDialog</class>
 <widget class="QDialog" name="CostDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Cost</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons/resources.qrc" >:/32/money.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Find:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="findlineEdit" />
       </item>
       <item>
        <widget class="QToolButton" name="cleartoolButton" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>x</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView" >
       <property name="selectionMode" >
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addpushButton" >
         <property name="text" >
          <string>Add</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelpushButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelpushButton</sender>
   <signal>clicked()</signal>
   <receiver>CostDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>228</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel" >
     <x>239</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cleartoolButton</sender>
   <signal>clicked()</signal>
   <receiver>findlineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>214</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel" >
     <x>175</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
